src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -165,4 +165,15 @@ def commented_unavailable
     flash.now[:error] = "Failed to create comment: This #{@commentable_type.name.downcase} does not exist anymore."
     render partial: 'layouts/webui/flash'
   end
+
+  def diff
```

Comment:
@hellcp-work can you please add some specs for this new method?

Reply:
I'm taking care of the specs. I will add it in a follow-up pr so this PR is not blocked